### PR TITLE
MM-2292: removed the double coding of SNOMED 27026000 and added display value 

### PR DESCRIPTION
**Pull request by Ashwin-nictiz**

### CODE REVIEW

**Comment by LuudSlagter** [comment]
Approved. Note however that this branch should be merged into `release-2.1.13` instead of `stable-2.x`.

**Review by LuudSlagter — APPROVED** [review]

File: Profiles - ZIB 2017/zib-IllnessPerception.xml [hunk 1]
```diff
@@ -110,16 +110,10 @@
         <coding>
           <system value="http://snomed.info/sct" />
           <code value="27026000" />
+          <display value="Insight"/>
```
Comment (pieter-edelman-nictiz) [inline Profiles - ZIB 2017/zib-IllnessPerception.xml]: Adding a required display value to profiles is undesired (and out of scope for this ticket). Please revert this change.

**Review by pieter-edelman-nictiz — CHANGES_REQUESTED** [review]

File: Profiles - ZIB 2017/zib-IllnessPerception.xml (hunk 1 above)
Comment (Ashwin-nictiz) [reply Profiles - ZIB 2017/zib-IllnessPerception.xml]: Fix!

**Review by pieter-edelman-nictiz — APPROVED** [review]
